utils/jsonHandler_test.go

```
@@ -0,0 +1,17 @@
+package utils
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestJsonHandler(t *testing.T) {
+	t.Run("should return json response", func(t *testing.T) {
+		assert.Equal(t, `{"age":"20","name":"John"}`, Json.ToJson(map[string]string{"name": "John", "age": "20"}))
+	})
+
+	t.Run("should return empty string when fails to marshal data", func(t *testing.T) {
```

- please fix the main method to return an error if we're not able to marshal the data instead of returning empty string.